src/classes/RD2_OpportunityService.cls

```diff
@@ -36,6 +36,8 @@
 */
 public with sharing class RD2_OpportunityService {
 
+    public static final Integer MIGRATION_INSTALLMENT_NUMBER = 0;
```

Comment:
Can you add a block comment here explaining what this represents?
Also, wondering if a var name change is warranted? Maybe something like `MIGRATION_INSTALLMENT_NUMBER_FLAG` or `MIGRATION_INSTALLMENT_CLOSED_FLAG`?

Reply:
Code is ready for review. Do we foresee the RD Installment Number being used to exclude (ignore) Opps by different processes not just data migration?

Reply:
I don't foresee that.